### ISSUE TITLE
Can't built libleak on CentOS 7 like system

### ISSUE THREAD

**Issue by vitodb** [WuBingzheng/libleak#17]
I tried to build libleak on a CentOS like system, but I can't figure out what package is supposed to provide the header
`backtrace.h`
I really appreciate any help to be able to use libleak on CentOS.

**Comment by WuBingzheng** [comment]
`libbacktrace` is built-in with GCC at my Ubuntu 16.04. It locates at `/usr/lib/gcc/x86_64-linux-gnu/5/libbacktrace.a`.

You can get and build it from `https://github.com/ianlancetaylor/libbacktrace` if your system does not have it.

**Comment by vitodb** [comment]
Thanks for the help.
Building and using `libbacktrace` as you pointed out, allowed me to build libleak on my system.

I tried to ran libleak on my code and I got an `ABORT` signal.
I figured out I have to increase this [array](https://github.com/WuBingzheng/libleak/blob/master/libleak.c#L194) to be able to use libleak to test my code.
Is it safe to increase that array? Or would this affect performances?
